### DISCUSSION TITLE
change point detection - # multivariable # offline # trend / seasonality

### DISCUSSION THREAD

**Discussion by ok123123123** [deepcharles/ruptures#237] [Q&A]
Hi,

I was wondering if you will be able to assist and direct me to the SOTA methods or papers about change point detection ?
I'm interested in particular with:
# multivariable / multi channels
# trend and seasonality change point detection
# offline.

thanks a lot,

Theodor

**Comment by deepcharles** [comment]
Hello, 

This question is not very clear or related to `ruptures` so I will close this issue.

Thanks

**Comment by ok123123123** [comment]
Hi,

I was looking for models to address this matter in ruptures library.
Would you be able to direct me to a forum which be suitable for this issue?

thanks.

‫בתאריך יום ב׳, 21 בפבר׳ 2022 ב-15:45 מאת ‪Charles T.‬‏ <‪
***@***.***‬‏>:‬

**Reply by deepcharles** [reply]
You can start with the review article associated with `ruptures` and work your way from there: Truong, C., Oudre, L., & Vayatis, N. (2020). Selective review of offline change point detection methods. Signal Processing, 167.

**Comment by ok123123123** [comment]
Thanks a lot.
I already covered it, was very useful.

בתאריך יום ג׳, 22 בפבר׳ 2022 ב-10:53 מאת Charles T. <
***@***.***>:

**Comment by ok123123123** [comment]
By the way, this article is by far the most informative, straightforward
and interesting I have read on this topic!

Thank you,

‫בתאריך יום ג׳, 22 בפבר׳ 2022 ב-12:37 מאת אור כץ ***@***.***‬‏>:‬

> Thanks a lot.
> I already covered it, was very useful.
>
> בתאריך יום ג׳, 22 בפבר׳ 2022 ב-10:53 מאת Charles T. <
> ***@***.***>:
>
>> You can start with the review article associated with ruptures and work
>> your way from there: Truong, C., Oudre, L., & Vayatis, N. (2020). Selective
>> review of offline change point detection methods. Signal Processing, 167.
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/deepcharles/ruptures/discussions/237#discussioncomment-2227002>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/AWZFOXZAPAKULFFIXYPN4FTU4NFJLANCNFSM5O6Q6HGA>
>> .
>> Triage notifications on the go with GitHub Mobile for iOS
>> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
>> or Android
>> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>>
>> You are receiving this because you authored the thread.Message ID:
>> ***@***.***>
>>
>

--
